README.md

```diff
@@ -1,2 +1,78 @@
-# terraform-provider-codeclimate
-CodeClimate provider for Terraform.
+Terraform CodeClimate Provider
+==================
+
+- Website: https://www.terraform.io
+- [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
+- [![Build Status](https://travis-ci.org/babbel/terraform-provider-codeclimate.svg?branch=master)](https://travis-ci.org/babbel/terraform-provider-codeclimate)
+- Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
+
+<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
```

Comment:
I assume we are not allowed to use the Terraform logo here

Reply:
@parabolic Do you know? I've taken it from your provider.